eng/common/scripts/Submit-PullRequest.ps1

```diff
@@ -86,12 +86,18 @@ else {
     $resp | Write-Verbose
     LogDebug "Pull request created https://github.com/$RepoOwner/$RepoName/pull/$($resp.number)"
   
+    $prOwnerUser = $resp.user.login
+
     # setting variable to reference the pull request by number
     Write-Host "##vso[task.setvariable variable=Submitted.PullRequest.Number]$($resp.number)"
 
+    # ensure that the user that was used to create the PR is not attempted to add as a reviewer
+    $usersCleaned = ($UserReviewers -split "," | % { $_.Trim() } | ? { $_ -ne $prOwnerUser })  -join ","
```

Comment:
Just to help avoid any future maintenance issues could you call https://github.com/Azure/azure-sdk-tools/blob/9e29cd69e0d0bec5f98e97e1c076ecc26c53154a/eng/common/scripts/Invoke-GitHubAPI.ps1#L12 (or a refactored version of it) to split these and then filter them as an array. The functions below will also take an array so no need to re-join them.

Reply:
I...can't say I agree here. Are you saying you want us to make another request to get PR information in the `Add-GitHubPullRequestReviewers` step?

It almost feels like we're moving from a generic usage to something more specific.

Reply:
![image](https://user-images.githubusercontent.com/45376673/111236631-91b18280-85b0-11eb-951f-9da7677e0731.png)

Reply:
Understood. Moving the array split code out of `Set-GitHubAPIParameters` and re-using in both locations.